src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * Storage class which handles loading / saving of task list from hard disk
+ *
+ */
+public class Storage {
+    private File file;
+    private String filepath;
+
+    /**
+     * Creates an instance of Storage.
+     * @param filepath
+     */
+    public Storage(String filepath) {
+        this.filepath = filepath;
```

Comment:
Since filepath will never change across the lifecycle of a program, it might be better to make Storage a static class instead, so that you won't have to pass an instance of Storage around every time you want to load or save a file